superpowers-desktop/conversion/README.md

```
@@ -0,0 +1,242 @@
+```bash
+./convert-all-skills.sh
+```
+
+**What it does:**
+- Converts all 20 skills from `../../skills/` directory
+- Organizes into categories (core, testing, debugging, collaboration, meta)
+- Copies example files (TypeScript, shell scripts)
+- Outputs to `../pro-setup/skills/` directory
+
+**Output structure:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)